smoketest.sh

```diff
@@ -125,7 +125,7 @@ runDemoApps() {
         --label io.cryostat.discovery="true" \
         --label io.cryostat.jmxHost="localhost" \
```

Comment:
Update here too.

Reply:
Not sure what you mean...

Reply:
Oh sorry, lets use `vertx-fib-demo-0`?

Reply:
Btw, Mergify seems to work as expected :))

Reply:
Opps, https://github.com/cryostatio/cryostat/pull/1646#issuecomment-1710598137

Reply:
It's close to working as expected but slightly different. I think what it's doing is good enough for now, since if reviewers have their GitHub notifications turned on, then they should get notified once they are tagged for review, as well as when the author comments with `/request_review` later, even if the label doesn't stick.

Reply:
SOunds good!